testsuite/integration-arquillian/servers/app-server/tomcat/tomcat9/pom.xml

```diff
@@ -26,19 +26,37 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>integration-arquillian-servers-app-server-tomcat9</artifactId>
-    <packaging>pom</packaging>
+    <packaging>jar</packaging>
     <name>App Server - Tomcat - Tomcat 9</name>
 
     <properties>
         <app.server.tomcat>tomcat9</app.server.tomcat>
-        
+
         <app.server.tomcat.groupId>org.apache.tomcat</app.server.tomcat.groupId>
         <app.server.tomcat.artifactId>tomcat</app.server.tomcat.artifactId>
         <app.server.tomcat.version>${tomcat9.version}</app.server.tomcat.version>
         <app.server.tomcat.unpacked.folder.name>apache-tomcat-${tomcat9.version}</app.server.tomcat.unpacked.folder.name>
-        
+
         <app.server.oidc.adapter.artifactId>keycloak-tomcat8-adapter-dist</app.server.oidc.adapter.artifactId>
         <app.server.saml.adapter.artifactId>keycloak-saml-tomcat8-adapter-dist</app.server.saml.adapter.artifactId>
```

Comment:
We should use keycloak-tomcat9-adapter-dist and keycloak-saml-tomcat9-adapter-dist which are not yet done. 

@hmlnarik can you suggest a way how to proceed with this one? I can see two possibilities we either add the adapter for tomcat9 within this PR or we don't add the tomcat9 test modules and wait for the adapters which should be done outside of this. (Or we can update this PR to use tomcat9 adapters and just don't include tests to the pipeline till the adapters are done) WDYT?